src/patterns/start-pages/index.md.njk

```diff
@@ -0,0 +1,53 @@
+---
+title: Start pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+If you’re building a digital service, you’ll need to provide a starting point for users on GOV.UK.
```

Comment:
very small thing, but the team building the service don't provide the starting point, GOV.UK do. However I can see this is current guidance, so maybe we dont need to change it right now

Reply:
I've made a minor tweak to fix this.